builder/packages_image.go

```diff
@@ -136,17 +136,17 @@ func (p *PackagesImageBuilder) determinePackagesLayerBaseImage(packages model.Pa
 }
 
 // NewDockerPopulator returns a function which can populate a tar stream with the docker context to build the packages layer image with
-func (p *PackagesImageBuilder) NewDockerPopulator(roleManifest *model.RoleManifest, forceBuildAll bool) func(*tar.Writer) error {
+func (p *PackagesImageBuilder) NewDockerPopulator(roles model.Roles, forceBuildAll bool) func(*tar.Writer) error {
```

Comment:
It needs to, otherwise we'd try to build all the packages even for one role (e.g. if we wanted gorouter, we don't need Java for UAA).

That _should_ be okay, since (for development) it should still build them based on existing package layers if they already contain the correct packages (via docker labels).